store/activities.js

```diff
@@ -2,31 +2,45 @@ const config = [
   {
     id: 'overview',
     path: 'overview',
+    icon: 'dashboard',
   },
   {
     id: 'content',
     path: 'content',
+    icon: 'document',
   },
   {
     id: 'depositDates',
     path: 'deposit-dates',
+    icon: 'calendar-check',
   },
   {
     id: 'plugins',
     path: 'plugins',
-  },
-  {
-    id: 'plugins/discovery',
-    path: 'plugins/discovery',
-  },
-  {
-    id: 'plugins/recommender',
-    path: 'plugins/recommender',
+    icon: 'puzzle',
+    subPaths: [
```

Comment:
It may be better to call nested paths `children`.

```suggestion
    children: [
```